README.md

```diff
@@ -1 +1,81 @@
 # Immutable Web Apps
+
+## Introduction
+
+_Immutable Web Applications_ is a framework-agnostic methodology for building and deploying static [single-page application](https://en.wikipedia.org/wiki/Single-page_application) that:
+
+- Minimizes risk and complexity of live releases.
+- Simplifies and maximizes caching.
+- Minimizes the need for servers and systems administration.
+- Enables continuous delivery through simple, flexible, atomic deployments.
+
+## Principles
+
+The methodology is based on the principles of ___strictly separating___:
+
+- Configuration from code.
+- Release tasks from build tasks.
```

Comment:
I think this principal might cover it but might be good to be even more explicit about the fact that assets are built only once and can be used in any environment.  Or maybe this is worth calling out in the 'Permabundle' section?

Reply:
I was planning on driving that point home in the 'workflow/development lifecycle' section that would be the next section and I have not written yet.

Reply:
I agree with Ryan's point here, especially since a deploy is really just putting an index.html somewhere. Not really sure how you could communicate that here without it becoming way more than just a bullet point. Something to think about though

Reply:
I want to take a wait-and-see approach to this point as we build out the document. Let's revisit it after the document is more fleshed out.

Reply:
I mean, one might start questioning why it is called 'immutable' if we don't emphasize 'assets are built only once'